tests/test_dbt.py

```diff
@@ -364,8 +364,9 @@ def test_local_diff(self, mock_diff_tables):
             _local_diff(diff_vars)
 
         mock_diff_tables.assert_called_once_with(
-            mock_table1, mock_table2, threaded=True, algorithm=Algorithm.JOINDIFF, extra_columns=tuple(column_set)
+            mock_table1, mock_table2, threaded=True, algorithm=Algorithm.JOINDIFF, extra_columns=ANY
```

Comment:
these were failing randomly due to set operations in the tested method causing a non-deterministic order